.github/CODEOWNERS

```
@@ -4,9 +4,9 @@
 .circleci/                    @napetrov @Alexsandruss @homksei @ahuber21 @ethanglaser
 
 # Docs
-*.md                          @maria-Petrova @napetrov @bdmoore1 @Alexsandruss
-doc/                          @maria-Petrova @napetrov @bdmoore1 @Alexsandruss
-requirements-doc.txt          @maria-Petrova @napetrov @bdmoore1 @Alexsandruss
+*.md                          @maria-Petrova @napetrov @emmwalsh @Alexsandruss
+doc/                          @maria-Petrova @napetrov @emmwalsh @Alexsandruss
+requirements-doc.txt          @maria-Petrova @napetrov @emmwalsh @Alexsandruss
```

```suggestion
requirements-doc.txt          @emmwalsh @Alexsandruss
```

Might be  there should be more people added here?

---

It would also automatically pick up everyone here as well: https://github.com/intel/scikit-learn-intelex/blob/main/.github/CODEOWNERS#L21, so probably wouldn't need to add to this line unless there is anyone desired not already in general requirements codeowners.

---

I think only Emm would be interested in doc updates but not general updates

---

Agreed, and I think setup matches this:
for general requirements updates, codeowners [here](https://github.com/intel/scikit-learn-intelex/blob/main/.github/CODEOWNERS#L21) are pulled in
for doc requirements, specifically, emm is pulled in based on line 9, along with general requirements codeowners above